class-pinterest-for-woocommerce.php

```diff
@@ -794,6 +849,21 @@ public static function is_connected() {
 		}
 
 
+		/**
+		 * Checks if connected by checking if we got a token in the db.
+		 *
+		 * @return boolean
+		 */
+		public static function is_business_connected() {
+
+			$account_data = self::get_setting( 'account_data' );
+			$is_partner   = isset( $account_data['is_partner'] ) ? (bool) $account_data['is_partner'] : false;
+
+			return $is_partner && self::is_connected();
```

Comment:
Why not check for is_connected early in the function, and avoid checking for specific account data?

This would save some checks from happening that wouldn't have any impact if we're not connected in the first place.

Reply:
Changed! thanks.